app/src/main/java/ru/prometeydev/movie/data/models/Actor.kt

```diff
@@ -0,0 +1,7 @@
+package ru.prometeydev.movie.data.models
+
+
+data class Actor(
+    val name: String,
+    val avatarDrawable: Int
```

Comment:
@DrawableRes

Reply:
Ok, I'll fix it

Reply:
I would not call it a fix, but improvement